conda/cli/activate.py

```diff
@@ -32,11 +35,27 @@ def prefix_from_arg(arg):
 
 
 def binpath_from_arg(arg):
-    path = join(prefix_from_arg(arg), 'bin')
-    if not isdir(path):
-        sys.exit("Error: no such directory: %s" % path)
+    if sys.platform == "win32":
+        path = [prefix_from_arg(arg),
+                join(prefix_from_arg(arg), 'Scripts'),
+                join(prefix_from_arg(arg), 'Library/bin'),
```

Comment:
The previous fix was not good enough, because `Library\bin` may not even exist in some environments. A better fix is 
```python
        path = [k for k in [prefix_from_arg(arg),
                            join(prefix_from_arg(arg), 'Scripts'),
                            join(prefix_from_arg(arg), join('Library','bin')),
                           ]
                  if os.path.isdir(k)
                ]
```